physical/raft/raft.go

```diff
@@ -1004,6 +1041,13 @@ func (b *RaftBackend) applyLog(ctx context.Context, command *LogData) error {
 		return errors.New("raft storage backend is not initialized")
 	}
 
+	for _, op := range command.Operations {
+		valueLen := len(op.Value)
+		if uint64(valueLen) > b.kvMaxValueSize {
+			return fmt.Errorf("%s; got %d bytes, max: %d bytes", physical.ErrValueTooLarge, valueLen, b.kvMaxValueSize)
+		}
+	}
+
 	commandBytes, err := proto.Marshal(command)
```

Comment:
Sorry for my vague comment earlier but I think we should be checking the size of the whole `commandBytes` value since this is actually what is being sent through raft. If we just check the size of each entry in a transaction we risk sending something that is n*max_kv_size through raft.

Since both Put and Transaction end up here I think we can check the size in just one location.

Reply:
Ahh, yes, that makes complete sense 👍 

But then the definition changes slightly -- its no longer a cap on the entry value size, but rather a cap on the _entire_ Raft entry/command. Correct me if I'm wrong.